<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_vis</class>
 <widget class="QWidget" name="f_vis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Визуализация</string>
  </property>
  <widget class="QGraphicsView" name="v_vis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>561</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_vis_forward">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>440</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Вперед</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_vis_backward">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_vis_about">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>10</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
